sycamore/tests/unit/transforms/test_merge_elements.py

```diff
@@ -0,0 +1,111 @@
+
+from transformers import AutoTokenizer
+
+from sycamore.data import Document, Element
+from sycamore.functions import filter_elements
+from sycamore.transforms.merge_elements import GreedyElementMerger
+
+
+class TestMergeElements:
+    dict0 = {
+        "doc_id": "doc_id",
+        "type": "pdf",
+        "text_representation": "text",
+        "binary_representation": None,
+        "parent_id": None,
+        "properties": {"path": "/docs/foo.txt", "title": "bar"},
+        "elements": {
+            "array": [
+                {
+                    "type": "UncategorizedText",
+                    "text_representation": "text1",
+                    "properties": {"filetype": "text/plain", "page_number": 1},
+                },
+                {
+                    "type": "UncategorizedText",
+                    "text_representation": "text2",
+                    "properties": {"filetype": "text/plain", "page_number": 2},
+                },
+                {
+                    "type": "Title",
+                    "text_representation": "title1",
+                    "bbox": [0.17606372549019608,
```

Comment:
These tests need to be readable, can you change the coordinates to be something more readable, same with text_representation if we can reduce the size

Reply:
ack